bug(jxa): project.status() returns 'active status' not 'active' in OF 4.8.8

## Bug

`project.status()` returns `"active status"` in OmniFocus 4.8.8 instead of `"active"`. The `normalizeStatus` helper in `project_create.js`, `project_get.js`, and `project_list.js` only handles `"on hold"` → `"on-hold"` and `"done"` → `"completed"`, but not `"active status"` → `"active"`.

## Repro

After creating a project, `status` in the response is `"active status"` rather than `"active"`.

Integration test failure:
```
AssertionError: expected 'active status' to be 'active'
```

## Acceptance Criteria

- [ ] `normalizeStatus` in all project scripts maps `"active status"` → `"active"` (and any other suffixed variants like `"on hold status"` → `"on-hold"`)
- [ ] `createProject + getProject round-trip` integration test passes — status is `"active"` not `"active status"`
- [ ] `listProjects filters by status` passes

## Affected files

- `src/scripts/jxa/project_create.js`
- `src/scripts/jxa/project_update.js`
- `src/scripts/jxa/project_get.js`
- `src/scripts/jxa/project_list.js`

--- a/src/scripts/jxa/project_create.js
+++ b/src/scripts/jxa/project_create.js
@@ -17,9 +17,12 @@ function run(argv) {
   ofApp.includeStandardAdditions = false;
 
   function normalizeStatus(raw) {
-    if (raw === "on hold") return "on-hold";
-    if (raw === "done") return "completed";
-    return raw;
+    // OmniFocus 4.8.8 returns verbose strings with a " status" suffix
+    // (e.g. "active status", "on hold status"). Strip for uniform handling.
+    const s = typeof raw === "string" ? raw.replace(/ status$/, "") : raw;
+    if (s === "on hold") return "on-hold";
+    if (s === "done") return "completed";
+    return s; // "active", "dropped"
   }
 
   function buildProject(proj) {

--- a/src/scripts/jxa/project_get.js
+++ b/src/scripts/jxa/project_get.js
@@ -15,9 +15,12 @@ function run(argv) {
   ofApp.includeStandardAdditions = false;
 
   function normalizeStatus(raw) {
-    if (raw === "on hold") return "on-hold";
-    if (raw === "done") return "completed";
-    return raw;
+    // OmniFocus 4.8.8 returns verbose strings with a " status" suffix
+    // (e.g. "active status", "on hold status"). Strip for uniform handling.
+    const s = typeof raw === "string" ? raw.replace(/ status$/, "") : raw;
+    if (s === "on hold") return "on-hold";
+    if (s === "done") return "completed";
+    return s; // "active", "dropped"
   }
 
   function buildProject(proj) {

--- a/src/scripts/jxa/project_list.js
+++ b/src/scripts/jxa/project_list.js
@@ -15,9 +15,12 @@ function run(argv) {
   ofApp.includeStandardAdditions = false;
 
   function normalizeStatus(raw) {
-    if (raw === "on hold") return "on-hold";
-    if (raw === "done") return "completed";
-    return raw; // "active", "dropped"
+    // OmniFocus 4.8.8 returns verbose strings with a " status" suffix
+    // (e.g. "active status", "on hold status"). Strip for uniform handling.
+    const s = typeof raw === "string" ? raw.replace(/ status$/, "") : raw;
+    if (s === "on hold") return "on-hold";
+    if (s === "done") return "completed";
+    return s; // "active", "dropped"
   }
 
   function buildProject(proj) {

--- a/src/scripts/jxa/project_update.js
+++ b/src/scripts/jxa/project_update.js
@@ -18,9 +18,12 @@ function run(argv) {
   ofApp.includeStandardAdditions = false;
 
   function normalizeStatus(raw) {
-    if (raw === "on hold") return "on-hold";
-    if (raw === "done") return "completed";
-    return raw;
+    // OmniFocus 4.8.8 returns verbose strings with a " status" suffix
+    // (e.g. "active status", "on hold status"). Strip for uniform handling.
+    const s = typeof raw === "string" ? raw.replace(/ status$/, "") : raw;
+    if (s === "on hold") return "on-hold";
+    if (s === "done") return "completed";
+    return s; // "active", "dropped"
   }
 
   function buildProject(proj) {

